### DISCUSSION TITLE
Tier 2.5 Rep Gains and Drop Rates are Woeful

### DISCUSSION THREAD

**Discussion by Vlaxitov** [chromiecraft/chromiecraft#4337] [Suggestions]
What I mean by that is I'm wondering if I'll end up just deleting my T2.5 tokens by the time we're into TBC content. The required idols to advance the quest are barely dropping. In addition to that, the amount of rep grind within the time we have left before we hit TBC bracket is unpleasantly pressing. I think these things should be looked at, considered, and adjusted accordingly so that there can be more incentive for an average player to clear the AQ25 raid more than 1-2 times for the novelty of it.

**Comment by balleny** [comment]
1) Nothing is stopping you from grinding additional bugs in Silithus, especially considering the reputation game from the 200 fragments turn in is already increased by factor 2,5 compared to classic
2) if you clear it on a regular basis, you will have enough artifacts to to get reputation relative to the amount of Idols dropping. If your guild lacks idols, advise your raid members to make sure checking the loot of the mobs to retrieve the Coffer Keys and dont forget to open those

**Reply by Vlaxitov** [reply]
Neither one of those suggestions improves the situation. The fragment turn in thing is over. You can do the quest line up to the broodlord's head and that's it. Its not like our guild isn't picking up every possible turn in including idols, we just never see them drop.

**Reply by balleny** [reply]
Please, just stop spreading misinformation before actually checking. Fragment turn in works just fine and you can use it to advance your reputation until you hit neutral. Myself and plenty of other people are farming fragments to increase their reputation in Silithus.

**Comment by Cooties-cc** [comment]
Comment from 2009/03/15 (Patch 3.0.9) :

_A full clear of AQ40, including all bosses, and all trash in your path (not going out of the way to kill mobs), yields ~14.500 reputation purely from trash, Twin Emperors and C'Thun. In addition, you get 9 Qiraji Lord Insignia's, granting 500 reputation for each turn-in, resulting in a total of 4500 reputation from Insignia's. Excluding possible Artifacts (1000 reputation per turn in) you'll have ~19.000 reputation with the Brood._ 

That amount is also increased if you're a human. If you start off hated with the rep, and don't do carapace turn ins, you need about 42k rep to hit neutral with the rep. Rep from killing things in AQ40 (and bosses in aq20) stops at the end of neutral (2999/3000). With that number, and most guilds clearing up to twins, that's about 3 runs or less (artifact drop chances) of AQ40 and you're at neutral. The highest rep you need for a piece of aq40 gear is to be at honored, which is another couple runs or less. 

The idea is that you should be saving all your insignias and artifacts until the end of being neutral, then turning them in when kills no longer give rep. If you've been wasting them blindly, then that's not the games fault. TBC 64 bracket will still be running aq40, as most classes will still be using around half of vanilla gear. That content is still weeks away.

**Reply by balleny** [reply]
those 14500 rep before turn ins is definitely not true on CC.
if you do a clear until Twins, you get approx 6k reputation if you are hated, excluding any turn ins

**Reply by Vlaxitov** [reply]
So you need be human, raid everything every week, and get lucky for 5-6 weeks out of the 6-7 weeks there is left before TBC. Also what balleny said, you're not getting 14500 a run even if you clear the entire instance. Its just unreasonable and when 60-64 happens why would anyone continue grinding out T2.5 only to replace it the very next bracket instead of grinding out the relevant reputations towards opening heroics in TBC content.

**Reply by balleny** [reply]
why did you raid MC when BWL was about to come 8 weeks later?
why did you run UBRS when MC was already announced?

To make the upcoming content even easier, for completions and achievements, bringing reputation factions to exalted, transmogs, etc, there are many valid reasons you are not considering cause you just want to find reasons to complain apparently.

Raiding 3-4 weeks should be sufficient to reach anything between neutral to honored even if you do clear only 6/9

**Reply by Vlaxitov** [reply]
@balleny The pre raid BIS and T1 scenarios aren't even in same ocean as T2.5 let alone the same boat. Pre raid BIS and T1 were very significant gains that were appreciated for a massive amount of content time comparatively. Going from T2 to T2.5 is minute gains that will be enjoyed for a minute amount of time. I highly doubt I'll have honored come my 4th run (we're currently at 8/9) but I guess we'll see. Even if I do get the rep there won't be nearly enough idols to go around to advance tier quests in ratio to the amount of tier tokens dropping.

Its kind of amusing that you'd use language like "you just want to find reasons to complain apparently," when I could have as lazily dismissed you by saying something like "you apparently just want to white knight the poorly planned rush through this content," instead of addressing your poor counterpoints. "Hey, maybe up these rates so that we can actually be rewarded for running this content in the short amount of time we have to run it before new content drops" is a legitimate suggestion. What is your actual interest in arguing that idol drop rates and rep gains shouldn't be upped?

**Comment by ldzetc** [comment]
After 2 weeks of killing all bosses minus C'thun, I'm still 28k/36k Hated.  
We didn't know about this rep and timeline before AQ40 was released.  It took longer to open the gates than we have to raid it.
Either up the rep and idol drops... or just remove the rep requirements and idol requirements from 2.5 turn in quests.

**Reply by ldzetc** [reply]
we've gotten 8-10 2.5 tokens and we've gotten enough idols for 3 turn ins?  
That's broken.

**Reply by balleny** [reply]
> We didn't know about this rep and timeline before AQ40 was released.

the timeline is essentially just so short, because even in the next bracket, the loot from AQ40 keeps being relevant

> It took longer to open the gates than we have to raid it.

nothing stops you from obtaining the currently best items available in the 64 bracket from AQ40

> and we've gotten enough idols for 3 turn ins?

Are you utilizing the given things? Do your raid members make sure to check if the loot of the trash mobs is assigned to them to check for coffer keys? I suspect you end up with 1 or 2 coffer keys only because the majority aint checking loot. Maybe consider raiding with trash loot on ffa, so one person can loot everything.

**Reply by Vlaxitov** [reply]
@balleny Why do you want to keep this content gated in a way that it forces people to continue grinding AQ instead of TBC factions?

**Comment by Vlaxitov** [comment]
I appreciate the added rep gains and upped drop rates of the idols. However, with the idols its still not nearly enough and there will be a lot of people going into TBC with T2.5 tokens rotting in their banks. The idol portion of the T2.5 quest line needs to be removed at this point for it to be reasonable.

**Reply by dennots** [reply]
Idol drop rate its still pretty low, we got 10 tier tokens on a full run and only 5 idols. Since you need 2 idols per token, the ratio on idols is 4 times lower than the tokens we get from bosses. Not even enough keys drop on a single run to open the coffers. Is this working as intended?

**Comment by Cooties-cc** [comment]
Not that it really matters that much, but t2.5 isn't the end all for every class. There are some that use it, and some that don't even touch it. I think it's been massively blown out of the water how much drama there's been over it. 

To start, you can check what blizz posted for their classic drop rates, which have been about avg 0.5-1.5% on mobs for idols. Coffers generally give 2 per (and have a 30min respawn timer). You can farm trash that respawns (given you didn't kill the boss linked with them) and get keys for idols / idol drops. All that being said, yes they have historically always been a low drop rate, but nothing that was to write home about. 

I've also personally gone through all the gear sets, and compared them to prior gear as an honest upgrade. Not a whole lot was really taken tbh. 

Bal druid : Chest
Feral tank : none
Resto druid : none
BM / Marks : none
Frost : Chest
Hpal : Chest
Prot pal : Chest
Ret : Chest
Disc : none
Spriest : none
Assn/combat : Whole set
Ele sham : Boots, chest, shoulders
Enhc sham : Head, chest, shoulders
Resto sham : Chest
Aff : Head, shoulders, chest
Arms : none
Prot war : Whole set

So realistically as honest upgrades/bis potential, you only need idols for 24 items. Of course it's not just 24 items, as you could have 2 prot war tanks, which is now 10 items instead of 5. Yet the idea still stands that idols are a rarity item that is best used for best potential, and not free candy you hand out.

**Reply by balleny** [reply]
I can mostly agree here.
Most people / groups complain about the low drop rate, but then only loot 2 or 3 coffer keys each run. Meanwhile, we remind our people to actually loot the trash and end up with more keys than coffers each run.

Also, small addition: Resto shaman uses the full set, 5pc bonus is extremely good and every decent guild will funnel the tokens into resto shamans first (esp. because their t2 is very bad)

**Reply by Cooties-cc** [reply]
Derailed subject, but I disagree with the full set t2.5 for resto shamans. The main takeaways are that the swap in sets ([from what I have laid out](https://eightyupgrades.com/set/7xuKNPhCaRSf1KYUnEBaVL)) ends up with : 

- -28sp
- +1.99% crit
- -14mp5

I do not agree that losing 28sp and 14mp5 for 2% crit and -0.4s cast on chain heal is worth at all.

**Comment by Amrx2** [comment]
Hi all, just to add that we have been taking a deep dive into all available data we have for these drop rates, including classic era servers, other server cores etc. What we have found is our drop rates on idols are already higher than all sources we can find. That would suggest a nerf would be in order, don't worry we aren't going to do that. 

Whilst you are getting 2 months of AQ40 almost before TBC opens, please don't forget that a good chunk of AQ40 gear will be massively relevant in the 64 bracket (which you have until February 2023). 

So as we currently already have the drops higher than they should be, we won't be making any further changes at this moment in time to these.

Cheers,
Construct
